### PR TITLE
Redirect non-admin users to 401 page when visiting /admin

### PR DESCRIPTION
**Pull request by bp323**

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: apps/timetable/apache/app.conf [hunk 1]
```diff
@@ -51,8 +51,9 @@
     </Directory>
 
     RewriteEngine On
-    RewriteRule ^/notfound$     <%= apacheConf.admin.shared %>/gh/errors/404.html
-    RewriteRule ^/unavailable$  <%= apacheConf.admin.shared %>/gh/errors/503.html
+    RewriteRule ^/accessdenied$     <%= apacheConf.admin.shared %>/gh/errors/401.html
+    RewriteRule ^/notfound$         <%= apacheConf.admin.shared %>/gh/errors/404.html
+    RewriteRule ^/unavailable$      <%= apacheConf.admin.shared %>/gh/errors/503.html
```
Comment (Coenego) [inline apps/timetable/apache/app.conf]: This will also need to be updated in the Puppet scripts

**Comment by Coenego** [comment]
Putting on hold as this should also be updated in the Puppet scripts

**Comment by bp323** [comment]
Removing `On Hold` label as puppet PR has been sent.

**Pull request merged**